notebooks/env.ipynb

```diff
@@ -0,0 +1,692 @@
+      "source": [
+        "### What is RLHive and how to install it"
+      ],
+      "metadata": {
+        "id": "EJutnqZVjNTu"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "xi-qOSphrCH7"
+      },
+      "source": [
+        "RLHive is a framework designed to facilitate research in RL. It provides the components necessary to run a full RL experiment, for both single agent and multi agent environments. It is designed to be readable and easily extensible, to allow users to quickly run and experiment with their own ideas. For installation, you can check [this notebook](https://colab.research.google.com/drive/11YirxgoVD7gjN02TdAeyFXOL1qH7Eydv?usp=sharing)."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "JWmMKQBPFoO8"
+      },
+      "outputs": [],
+      "source": [
+        "!pip install ruamel.yaml\n",
+        "!pip install pyglet\n",
+        "!pip install git+https://github.com/chandar-lab/RLHive.git@dev\n",
+        "\n",
+        "!python -m pip install pyvirtualdisplay\n",
+        "!pip3 install box2d\n",
+        "!sudo apt-get install xvfb"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import torch\n",
+        "import hive\n",
+        "from hive.utils.registry import registry\n",
+        "from hive.envs.base import BaseEnv\n",
+        "from hive.envs.gym_env import GymEnv\n",
+        "from hive.envs.env_spec import EnvSpec\n",
+        "from ruamel import yaml\n",
+        "import sys\n",
+        "import os\n",
+        "import os.path\n",
+        "import numpy as np\n",
+        "import matplotlib.pyplot as plt\n",
+        "from pyvirtualdisplay import Display\n",
+        "%matplotlib inline"
+      ],
+      "metadata": {
+        "id": "VKastN5fSqsP"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### How to install environments\n",
+        "\n",
+        "RLHive currently supports the following environments:\n",
+        "\n",
+        "\n",
+        "\n",
+        "*   Gym classic control\n",
+        "*   Atari\n",
+        "* Minatar (simplified Atari)\n",
+        "* Minigrid (single-agent grid world)\n",
+        "* Marlgrid (multi-agent)\n",
+        "* Pettingzoo (multi-agent)\n",
+        "\n",
+        "To install Gym, you could simply run `pip install gym==0.21.0`. You can also install dependencies necessary for the environments that RLHive comes with by running `pip install rlhive[<env_names>]` where `<env_names>` is a comma separated list made up of `atari`, `gym_minigrid`, and `pettingzoo`.\n",
+        "\n",
+        "Minatar and Marlgrid are also supported, but must be installed separately.\n",
+        "\n",
+        "* To install Minatar, run `pip install MinAtar@git+https://github.com/kenjyoung/MinAtar.git@8b39a18a60248ede15ce70142b557f3897c4e1eb`\n",
```

Comment:
I think MinAtar could be reached directly via Gym (see [here](https://github.com/kenjyoung/MinAtar#openai-gym-wrapper)). Could you check if true, then remove this, please?